pom.xml

```diff
@@ -3,215 +3,10 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.ascendix.salesforce</groupId>
     <artifactId>salesforce-jdbc</artifactId>
-    <version>1.2-SNAPSHOT</version>
-
-    <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>

```

Comment:
no need to duplicate compiler, sourceEncoding, dependency versions and plugin version **properties** in sub-module POMs, while you can have it centralized in parent pom.
Also you can use `<dependencyManagement>` in parent. So you don't need to specify version in modules